### PR TITLE
feat: implement virtual scrolling to reduce DOM size

### LINKED ISSUE #18: Deal with massive amounts of frames
Massive amounts of frames cause the extension to crash, because too much DOM nodes are rendered.
Please see for further explanation:
https://web.dev/dom-size/

## Motivation

We are working on a real-time application with dozens of frames send each minute.
This quickly stacks up and crashed this extension.

## Desired solution

Only render that DOM which is visible in the UI to the user, called virtual scrolling or windowing.
https://web.dev/dom-size/#how-to-optimize-the-dom-size

## Considered alternatives

None

### PR DESCRIPTION
**Pull request by AndyOGo**
_[One line description of your change]_

This PR utilizes [`react-window`](react-window.now.sh/) to improve DOM size as suggested here:
https://web.dev/dom-size/#react

### Motivation:

_[Explain here the context, and why you're making that change. What is the problem you're trying to solve.]_

This extension is great, but with massive amount of frames it crashed.
fixes #18 

### Modifications:

_[Describe the modifications you've done.]_

I installed two NPM packages and replaced the frames list with virtual scrolling.

### Result:

_[After your change, what will change.]_

RSocket-Frame-Viewer-virtual-scrolling-Jan-14-2022 13-21-11-compressed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "@types/react-dom": "^16.9.10",
     "@types/react-fontawesome": "^1.6.4",
     "@types/react-inspector": "^4.0.1",
+    "@types/react-virtualized-auto-sizer": "^1.0.1",
```
Comment (AndyOGo) [inline package.json:20]: Do I need to bump the version manually or does this happen upon release?

- [x] Check before merging

Comment (SerCeMan) [reply package.json:20]: I'll bump the version on release 👍

**Review by SerCeMan — APPROVED** [review]
Thanks, @AndyOGo for the contribution! 🚀

File: src/viewer/App.tsx [hunk 2]
```diff
@@ -412,6 +415,24 @@ export const App = observer(({store}: { store: AppStateStore }) => {
   }
   const {frames, activeFrame} = connection;
   const active = frames.find(f => f.id === activeFrame);
+
+  const Row = ({ index, style, data }: ListChildComponentProps) => {
+    const frame = data[index];
```
Comment (AndyOGo) [inline src/viewer/App.tsx:420]: @SerCeMan 
Sorry I just realized, that I broke that line.

It should access `frames[index]` not `data[index]` (the later would only work if [`itemData` prop would be set on `<FixedSizeList />`](https://react-window.vercel.app/#/api/FixedSizeList))

I provided a fix as part of #21

**Pull request merged**